### PR TITLE
Fix/no extraneous dependencies

### PR DESCRIPTION
**Pull request by MKaHead**
fix that the `no extraneous dependencies` eslint rule work correctly and remove unnecessary eslint disable because the rule don't know about our lerna packages.

# Merge Checklist:
- [x] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch-webhub-cloud/pattern-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [x] Tests are sufficient.
- [x] Documentation is up to date.
- [x] Changelog is up to date.
- [x] Typescript typings for properties are up to date.
- [x] Designers approved changes or no approval needed.
- [x] Dependencies to other components still work.
- [x] Tested on Windows

### CODE REVIEW

File: .eslintrc.cjs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+          '**/react/*.{js,jsx}',
+          '**/05-utils/**/*.{js,jsx}',
+          '**/src/**/*.react.js',
+          '**/demo/**',
+          '**/ui.test.js',
+          '**/unit.test.{js,jsx}',
+          '**/*test.accessor.js',
+          '**/*ui.test.js',
+        ],
+        packageDir: [__dirname, ...lernaJSON.packages],
```
Comment (MKaHead) [inline .eslintrc.cjs:54]: Maybe there need to test this on Windows

Comment (markus-walther) [reply .eslintrc.cjs:54]: Yes, that would be good. Added to checklist...

File: src/components/10-atoms/link/index.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // TODO fix that stuff
```
Comment (markus-walther) [inline src/components/10-atoms/link/index.js]: That 'stuff' is fixed in this PR, so remove the comment :-)

File: src/components/20-molecules/file-upload/index.js [hunk 3]
```diff
@@ -73,6 +71,7 @@ const findAndManipulate = (
     // N.B. this is the only case that's *independent* of file-found status
     if (_append && _append.has(where)) appends.push(where);
     // file not found?
+    // eslint-disable-next-line no-continue
```
Comment (markus-walther) [inline src/components/20-molecules/file-upload/index.js:74]: I like 'continue'. Wouldn't mind if there was no Eslint rule against it....

File: src/components/30-organisms/footer/ui.accessor.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-/* eslint-disable import/no-extraneous-dependencies */
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (markus-walther) [inline src/components/30-organisms/footer/ui.accessor.js]: ummm... why do we need it back here?

**Review by markus-walther — CHANGES_REQUESTED** [review]
Hi Marco,

mostly fine work - few nitpicks, if you will.

Cheers, Markus

File: package.json [hunk 5]
```diff
@@ -14,8 +14,8 @@
     "start": "npm run storybook",
     "storybook": "start-storybook -p 6006 --no-version-updates",
     "build-storybook": "build-storybook",
-    "test": "npm run test-lint && prettier --check . && vitest run && node ./scripts/build.test.js && npm run test-ui",
-    "test-lint": "eslint . --ext .js,.jsx --ignore-path .gitignore",
+    "test": "npm run lint && prettier --check . && vitest run && node ./scripts/build.test.js && npm run test-ui",
+    "lint": "eslint . --ext .js,.jsx --ignore-path .gitignore",
```
Comment (markus-walther) [inline package.json:18]: Sounds much better.

File: src/components/10-atoms/link/index.js [hunk 6]
```diff
@@ -1,5 +1,5 @@
...
-/* eslint-disable import/no-extraneous-dependencies */
+
```
Comment (markus-walther) [inline src/components/10-atoms/link/index.js]: :-)

**Review by markus-walther — DISMISSED** [review]
Marco, LGTM. --Markus

**Review by markus-walther — APPROVED** [review]
Marco, now it's perfect! --Markus

**Pull request merged**